src/ir/op.cc

```diff
@@ -122,6 +124,19 @@ TVM_REGISTER_GLOBAL("ir.RegisterOpAttr")
       }
     });
 
+TVM_REGISTER_GLOBAL("ir.RegisterOpLowerIntrinsic")
+    .set_body_typed([](String name, TVMFunctionHandle f, String target = "default", int plevel = 10,
+                       int can_override = 0) {
```

Comment:
Forget about TVMFunctionHandle, which is only used in the C API but nowhere else :-)

Let's embrace the PackedFunc magic! It is a nicer wrapper of a function from either C++ or python (or potentially other languages).

BTW, don't need to set default arguments of a lambda because they are not used any way.

```suggestion
    .set_body_typed([](String name, PackedFunc f, String target, int plevel, bool override_) {
```